src/base/utils/net.h

```diff
@@ -47,6 +49,12 @@ namespace Utils
         bool isLoopbackAddress(const QHostAddress &addr);
         bool isIPInRange(const QHostAddress &addr, const QList<Subnet> &subnets);
         QString subnetToString(const Subnet &subnet);
+
+        const int MaxSSLFileSize = 1024 * 1024;
```

Comment:
It should be MAX_SSL_FILE_SIZE to be consistent with currently used (de-facto) coding style.

Reply:
Done.